frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -1,504 +1,14 @@
-import * as _ from 'lodash';
-import {
-  DashboardsOverviewHealthResourceSubsystem,
-  HorizontalNavTab,
-  ModelDefinition,
-  ModelFeatureFlag,
-  Plugin,
-  ResourceTabPage,
-  RoutePage,
-  ResourceDetailsPage,
-  DashboardsOverviewResourceActivity,
-  CustomFeatureFlag,
-  ProjectDashboardInventoryItem,
-  ResourceListPage,
-} from '@console/plugin-sdk';
-import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
-import { referenceForModel } from '@console/internal/module/k8s';
 import * as models from './models';
-import * as mockModels from './mock-models';
-import { getCephHealthState } from './components/dashboards/persistent-internal/status-card/utils';
-import { isClusterExpandActivity } from './components/dashboards/persistent-internal/activity-card/cluster-expand-activity';
-import { WatchCephResource } from './types';
-import {
-  detectOCS,
-  detectOCSSupportedFeatures,
-  detectRGW,
-  CEPH_FLAG,
-  MCG_FLAG,
-  OCS_FLAG,
-  ODF_MANAGED_FLAG,
-  detectManagedODF,
-  detectComponents,
-  FEATURES,
-  RGW_FLAG,
-  ODF_ADMIN,
-} from './features';
 import { ODF_MODEL_FLAG } from './constants';
-import { getObcStatusGroups } from './components/dashboards/object-service/buckets-card/utils';
 
-type ConsumedExtensions =
-  | ModelFeatureFlag
-  | HorizontalNavTab
-  | ModelDefinition
-  | DashboardsOverviewHealthResourceSubsystem<WatchCephResource>
-  | RoutePage
-  | CustomFeatureFlag
-  | ResourceDetailsPage
-  | ResourceTabPage
-  | DashboardsOverviewResourceActivity
-  | ProjectDashboardInventoryItem
-  | ResourceListPage;
-
-const apiObjectRef = referenceForModel(models.OCSServiceModel);
-const blockPoolRef = referenceForModel(models.CephBlockPoolModel);
-const storageSystemGvk = referenceForModel(models.StorageSystemModel);
-
-const OCS_MODEL_FLAG = 'OCS_MODEL';
-
-const plugin: Plugin<ConsumedExtensions> = [
-  {
-    type: 'ModelDefinition',
-    properties: {
-      models: _.values(models),
-    },
-  },
-  {
-    type: 'FeatureFlag/Model',
-    properties: {
-      model: models.OCSServiceModel,
-      flag: OCS_MODEL_FLAG,
-    },
-  },
+const plugin = [
   {
     type: 'FeatureFlag/Model',
     properties: {
       model: models.StorageSystemModel,
       flag: ODF_MODEL_FLAG,
     },
   },
```

Comment:
I don;t think this is needed... where is it used in odf-console (and if used, are we not setting already in odf-console) ??

Reply:
Are you asking about ODF_MODEL_FLAG?  When i remove this ODF_MODEL_FLAG flag, i could see the "Data Foundation" link visible on the left side even when we do not have ODF operator installed.

Reply:
did u check whether odf-console is setting this flag or not ?

Reply:
odf-console is not setting the default value to false where as its setting to true only.

Reply:
odf-console will only be present when ODF operator is present...
once odf-console is there, it will set flag to true...

Reply:
So we can keep the flag for now right and can remove it once we have fix in place at odf-console ?

Reply:
I don;t think doing that will cause the issue which u mentioned above... plz make sure u r running local servers correctly...